pai-management/bootstrap/prometheus/node-label.sh.template

```diff
@@ -24,4 +24,7 @@ kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} prometheus=true
     {% if 'node-exporter' in machinelist[ host ] and machinelist[ host ][ 'node-exporter' ] == 'true' %}
 kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} node-exporter=true
     {% endif %}
+    {% if 'watchdog' in machinelist[ host ] and machinelist[ host ][ 'watchdog' ] == 'true' %}
```

Comment:
Please be sure, this label could be work in current master branch. Note: the configuration style has been changed. Please modify the mock cluster-object-model.

Reply:
test, when set  watchdog: "true" at cluster-configuration.yaml. it will deploy. This file update this config